eng/build.yml

```diff
@@ -168,3 +168,12 @@ phases:
         publishLocation: Container
       continueOnError: true
       condition: always()
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/7.0.1xx'))) }}:
+  - template: /eng/common/templates/job/onelocbuild.yml
```

Comment:
This will try to find JSON and WXL files to translate but will miss https://github.com/dotnet/installer/blob/main/src/redist/targets/packaging/windows/clisdk/LCID/1033/bundle.wxl because it's already in a locale-specific directory. Suggest copying that file to a parent folder. For example, dotnet/aspnetcore has both https://github.com/dotnet/aspnetcore/blob/main/src/Installers/Windows/WindowsHostingBundle/thm.wxl and https://github.com/dotnet/aspnetcore/blob/main/src/Installers/Windows/WindowsHostingBundle/1033/thm.wxl. The first is an input WiX should ignore and the second is the "localized" file included in the English installer.

@jonfortescue the dotnet/runtime team has a similar layout. Perhaps an option to find the `1033` file when nothing else exists would make sense❔ Could also unconditionally look first for `.../1033/radishes.wxl` and put that in the loc project unless `rashishes.wxl` exists in a parent folder. (May need to be careful w/ placement w/in the loc project but that's not a big deal, right❔)

Reply:
So if I just create an additional version of the 1033 version one level up, it'll find that file (and we just have to keep it in sync with the english version from now on)?

Reply:
@jonfortescue should confirm but the way WXL loc works for ASP.NET is we only touch the English file that's **not** in the 1033 folder. That's the input the process gives to the loc team. Later, loc handback PRs update all of the LCID folders, including the 1033 one.

Reply:
Precisely correct!